lessons/flow.md

```diff
@@ -112,21 +149,207 @@ No errors!
 
 Congratulations, you are ready to develop using Flow 🚀.
 
-### Prepare your files for Flow 
+### 2.3 Prepare your files for Flow 
 
 In order to tell the compiler which file it should check, you need to mark it as a Flow file by placing the following line at the top of the file:
 
 ```js
 // @flow
 ```
 
-Then the compiler will analise all these files at compile time to ensure consistency across the type definitions. 
+Then the compiler will analyse all these files at compile time to ensure consistency across the type definitions. 
 
 ## 3. Writing Flow code
+
+### 3.1 [Primitive types](https://flow.org/en/docs/types/primitives/)
+
+#### 3.1.1 Literal vs. wrapper types
+
+In Flow there are types for the literal values that are lowercase, and types for the wrapper objects which are 
+capitalized. Therefore, the type `Number` is not the same as the type `number`. The full list of primitive types is:
+
+* [`Booleans`](https://flow.org/en/docs/types/primitives/#toc-booleans)
```

Comment:
I feel like these should be lower cased given the previous sentence as it might cause confusion.